gulpfile.js

```diff
@@ -41,6 +56,9 @@ gulp.task('babel-client', ['update-modules'], function() {
         .pipe(source('start.js'))
         .pipe(gulp.dest('.'))
         .pipe(notify("start.js built!!"));
+
+
+    return merge(main, start);
```

Comment:
Not sure this code will work correctly with sourcemaps, merge latest changes and check